packages/webapi/src/index.ts

```diff
@@ -0,0 +1,25 @@
+import * as express from 'express';
+import { Store, HasId } from '@raincatcher/store';
+import WebApiConfig from './config';
+
+/**
+ * Raincatcher webapi service module
+ * 
+ * Create codeless API using express.js
+ * 
+ * @param {Store} store - storage implementation 
+ * @param {WebApiConfig} config - module configuration
+ * @return {express.Router} router that can be mounted in top level application
```

Comment:
We can omit type annotations from the jsdocs, typedoc should determine the right types. Adding them here even if they work just adds another place that needs to be updated on a change